### PR TITLE
docs: Describe how vfio devices can be passed

### PR DESCRIPTION
**Pull request by amshinde**
VFIO device groups can now be passed with --device.
Describe the setup steps required.

Signed-off-by: Archana Shinde <archana.m.shinde@intel.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -258,3 +258,79 @@ Ability to hotplug block devices has been added, which makes it possible to use
...
+kernel by verifying `CONFIG_VFIO_NOIOMMU` is not in the kernel config. If it is set,
+you will need to rebuild your kernel.
+
+The following kernel configs need to be enabled:
+```
+CONFIG_VFIO_IOMMU_TYPE1=m 
+CONFIG_VFIO=m
+CONFIG_VFIO_PCI=m
+```
+
```
Comment (mcastelino) [inline README.md]: @amshinde you also need to add the requirement that iommu needs to be enabled at boot 
```
intel_iommu=on
```
Most distro's do not enable iommu by default

File: README.md [hunk 2]
```diff
@@ -258,3 +258,79 @@ Ability to hotplug block devices has been added, which makes it possible to use
...
+2. Identify BDF(Bus-Device-Function) of the PCI device to be assigned.
+
+```
+01:00.0 Ethernet controller: Intel Corporation Ethernet Controller 10-Gigabit X540-AT2 (rev 01)
```
Comment (mcastelino) [inline README.md]: @amshinde add another description on how to go from physical device to PCI bdf. As most people will have a device driver loaded against the device.

example using

```
ethtool -i eth0
```

File: README.md [hunk 3]
```diff
@@ -258,3 +258,79 @@ Ability to hotplug block devices has been added, which makes it possible to use
...
+```
+
+8. Start a Clear Containers container passing the VFIO group on the docker command line.
+
+```
+docker run -it --device=/dev/vfio/16 centos/tools bash
+```
+
+9. Running `lspci` within the container should show the device among the 
+PCI devices.
```
Comment (mcastelino) [inline README.md]: @amshinde add another point saying that the driver for the device needs to be present within the CC kernel (or they need to rebuild the same)

**Comment by amshinde** [comment]
@mcastelino Comments addressed

**Comment by sameo** [comment]
LGTM

**Pull request merged**